metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java

```diff
@@ -27,7 +28,7 @@
 @Slf4j
 @Component
 @Conditional(MetadataChangeLogProcessorCondition.class)
-@Import({ UpdateIndicesHook.class, IngestionSchedulerHook.class })
+@Import({ UpdateIndicesHook.class, IngestionSchedulerHook.class, KafkaEventConsumerFactory.class })
```

Comment:
Funny that we had to add this...

Reply:
It was there before, but was removed in the managed ingestion pr.